Modules/Project/Resources/views/edit.blade.php

```
@@ -51,8 +54,19 @@
 
         updateProjectForm: async function(formId) {
             let formData = new FormData(document.getElementById(formId));
-            let response = await axios.post('{{ route('project.update', $project) }}', formData);
-            alert('Project information updated successfully');
+            await axios.post('{{ route('project.update', $project) }}', formData)
+            .then((response) => {
+                $('#edit-project-errors').addClass('d-none')
+                alert('Project information updated successfully');
+            })
+            .catch((error) => {
+                let errors = error.response.data.errors;
+                $('#edit-project-error-list').empty()
+                for (error in errors) {
+                    $('#edit-project-error-list').append("<li class='text-danger ml-2'>" + errors[error] + "</li>");
+                }
+                $('#edit-project-errors').removeClass('d-none')
+            })
```

show and hide error messages